pkg/networkmanager/network-main.jsx

```diff
@@ -138,6 +138,7 @@ export const NetworkPage = ({ privileged, operationInProgress, usage_monitor, pl
 
     const actions = privileged && (
         <>
+            <NetworkAction buttonText={_("Add WireGuard")} type='wireguard' />
             <NetworkAction buttonText={_("Add bond")} type='bond' />
             <NetworkAction buttonText={_("Add team")} type='team' />
             <NetworkAction buttonText={_("Add bridge")} type='bridge' />
```

Comment:
I think at some point we are going to accumulate too many buttons here -- there's going to be more, for e.g. OpenVPN. Perhaps we need to categorize them, into "Add virtual device" (bond, bridge, team, VLAN) and "Add VPN" (WireGuard, OpenVPN, etc.), or turn this into a Kebab?

I think that's a question for @garrett . For continuing the development of this PR this is fine of course.

Reply:
Yep, too many buttons.

I do like the idea of "Add virtual device" and "Add VPN". However...

The issues: 
- how common is each task?
- adding a switcher to unrelated tasks (bond/bridge/VLAN) isn't so great
- bond in particular is quite verbose
- having everything under a ... menu kind of hides away what Cockpit can do
- there are [split buttons](https://staging.patternfly.org/components/menus/menu-toggle#split-button-toggle-with-action), but those (again) would be for a [default action and other _related_ actions](https://staging.patternfly.org/components/menus/dropdown/design-guidelines#split-button-with-actions).

Should we do "Add VPN" (with a switcher of VPN types in the modal, once we have more than just Wireguard) and a kebab with the rest? :shrug: 

(Also: We should probably default to Wireguard, once we have more VPN types, as Wireguard the most "native" and flexible VPN.)

Right now, we might want to keep this in mind and just tack on an "Add VPN" for this PR and handle the rest in a follow-up. What do you think? (Alternative is to "Add VPN" and move the rest to a kebab menu in this PR, I think.)